assay_configs/HRD/eggd_conductor_dias_HRD_config_v1.0.0.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "Config for HRD assay",
+    "assay": "HRD",
+    "assay_code": "-9197|-10021",
+    "version": "1.0.0",
+    "details": "Includes 'demultiplexing and multiQC",
+    "changelog": {
+        "v1.0.0": "Initial working version"
+    },
+    "demultiplex": true,
+    "users": {
+        "org-emee_1": "CONTRIBUTE"
+    },
+    "executables": {
+        "applet-FvyXygj433GbKPPY0QY8ZKQG": {
+            "name": "multi_fastqc",
+            "details": "Takes all fastqs of all samples, starts one job which runs fastqc in sub jobs for each fastq",
+            "url": "https://github.com/eastgenomics/eggd_fastqc",
+            "analysis": "analysis_1",
+            "per_sample": false,
+            "process_fastqs": true,
+            "inputs": {
+                "fastqs": "INPUT-R1-R2"
+            },
+            "output_dirs": {
+                "applet-FvyXygj433GbKPPY0QY8ZKQG": "/OUT-FOLDER/APP-NAME"
+            }
+        },
+     "app-GF3K4Qj4bJyvpzx055V3G8q7": {
+            "name": "MultiQC",
+            "details": "MultiQC app to generate aggregated run level QC report",
+            "url": "https://github.com/eastgenomics/eggd_multiqc",
+            "analysis": "analysis_2",
+            "per_sample": false,
+            "process_fastqs": false,
+            "depends_on": [
+                "analysis_1"
+            ],
+            "inputs": {
+                "project_for_multiqc": "INPUT-dx_project_name",
+                "primary_workflow_output": "INPUT-parent_out_dir",
+                "calc_custom_coverage": false,
+                "multiqc_docker": {
+                    "$dnanexus_link": "file-GF3PxgQ433Gqv1Q029Gjzjfv"
```

Comment:
should we include project ID here like in tso500 config to make sure it takes the 001 version?

Reply:
This will not be needed when the copy of the file gets copied into 001_reference and deleted from the development project where testing happened  
<!-- Reviewable comment -OEUJyonDdZrpVjlmjTi:bgdasdr -->
<!-- Sent from Reviewable.io -->